vscode-dotnet-runtime-library/src/Acquisition/InstallScriptAcquisitionWorker.ts

```diff
@@ -25,7 +25,7 @@ export class InstallScriptAcquisitionWorker implements IInstallScriptAcquisition
         const scriptFileEnding = os.platform() === 'win32' ? 'ps1' : 'sh';
         const scriptFileName = 'dotnet-install';
         this.scriptFilePath = path.join(__dirname, 'install scripts', `${scriptFileName}.${scriptFileEnding}`);
-        this.webWorker = new WebRequestWorker(extensionState, eventStream, this.scriptAcquisitionUrl + scriptFileEnding, scriptFileName);
+        this.webWorker = new WebRequestWorker(extensionState, eventStream, this.scriptAcquisitionUrl + scriptFileEnding, scriptFileName + scriptFileEnding);

```

Comment:
Ahh good catch! 👍

Reply:
Thanks, do the changes look good from a releasing standpoint? I talked with Sarah about how to make a new release but wanted to check if you knew of anything.